src/components/Dropdown/styles.tsx

```diff
@@ -0,0 +1,22 @@
+import type { CSSObjectWithLabel, ControlProps, GroupBase } from 'react-select';
+import type { SelectOption } from './Dropdown.types';
+
+// Better align Select wih CFPB styles
+export const extendedSelectStyles = {
+  control: (
+    base: CSSObjectWithLabel,
+    state: ControlProps<SelectOption, boolean, GroupBase<SelectOption>>
+  ): CSSObjectWithLabel => ({
+    ...base,
+    borderColor: state.isFocused ? '#0072ce' : base.borderColor,
```

Comment:
It'd be ideal if we could use the [`@pacific`](https://github.com/cfpb/design-system/blob/6f3c87c49283ffb3cd3437b32c0f2e02667ab658/packages/cfpb-core/src/brand-colors.less#L68) variable name here instead of hardcoding the color but I haven't had luck finding a way to do that. 🤷

Reply:
We might be able to use something like [less-vars-to-js](https://www.npmjs.com/package/less-vars-to-js) to parse out the [brand-colors](https://github.com/cfpb/design-system/blob/6f3c87c49283ffb3cd3437b32c0f2e02667ab658/packages/cfpb-core/src/brand-colors.less) variables into a JS object that we could reference.  
 
This could be a one-off solution in Design Stories or possibly integrated into the Design System as part of a script that generates a JS version of the variables for broader reuse.

Reply:
Cool I'll open a low-priority ticket about it.